.gitlab-ci.yml

```
@@ -0,0 +1,117 @@
+.hykkt_script_template:
+ script:
+    - |
+      set -xv
+      export WORKDIR="$HOME/gitlab/$CI_JOB_ID"
+      if [[ ! -d "$WORKDIR" ]]; then
+        mkdir -p "$WORKDIR"
+        cp -R ./* "$WORKDIR"
+      fi
+      cd $WORKDIR
+      [ -f output ] && rm output
+      touch output
+      tail -f output &
+      tailpid=$!
+      sinfo -l | grep $SLURM_Q
+      #test build script
+      jobid=$(sbatch --export=ALL -A EXASGD --gres=gpu:1 --ntasks=1 -p $SLURM_Q -o output -e output -t 60:00 $WORKDIR/buildsystem/build.sh $SCRIPT_ARGS)
+      
+      export jobid=$(echo $jobid | cut -f4 -d' ')
+      echo $jobid > "$WORKDIR/jobid"
+      res=1
+      while :;
+      do
+        if [[ "$(awk 'BEGIN{i=0}/BUILD_STATUS/{i++}END{print i}' output)" != "0" ]]; then
+          kill $tailpid
+          echo 'Last tail of build output:'
+          tail -n 200 output
+          res=$(grep BUILD_STATUS output | tail -n 1 | cut -f2 -d':')
+          break
+        fi
+        sleep 30
+      done
+      rm "$WORKDIR/jobid"
+      echo "Finished job with code $res"
+      exit $res
+
+
+.pnnl_tags_template:
+  tags:
+    - k8s
+    - ikp
+    - exasgd
+    - marianas
+    - deception
+
+.pnnl_nonhpc_tags:
+  tags:
+    - k8s
+    - ikp
+    - exasgd
+    - basic
+
+hykkt-test-deception:
+  stage: test
+  needs: []
+  variables:
+    SLURM_Q: "a100_shared"
+    MY_CLUSTER: "deception"
+  extends:
+    - .pnnl_tags_template
+    - .hykkt_script_template
+
+hykkt-test-newell:
+  stage: test
+  needs: []
+  variables:
+    SLURM_Q: "newell_shared"
+    MY_CLUSTER: "newell"
+  extends:
+  - .pnnl_tags_template
+  - .hykkt_script_template
+
+
+SVC Account Cleanup:
+  stage: .pre
+  extends:
+   - .pnnl_nonhpc_tags
```

```suggestion
   - .pnnl_tags_template
```

This is the same change as in #5